tests/sentry/tasks/test_delete_seer_grouping_records.py

```diff
@@ -33,50 +32,43 @@ def test_delete_seer_grouping_records_by_hash_batches(
             "args": [project_id, hashes, 100]
         }
 
-    @patch("sentry.tasks.delete_seer_grouping_records.logger")
-    def test_call_delete_seer_grouping_records_by_hash_simple(self, mock_logger):
+    @patch(
+        "sentry.tasks.delete_seer_grouping_records.delete_seer_grouping_records_by_hash.apply_async"
```

Comment:
Notice that we are mocking the `apply_async` call which is used for scheduling the task.